ReplayKitExample/ReplayKitExample/ReplayKitVideoSource.swift

```diff
@@ -145,6 +253,59 @@ class ReplayKitVideoSource: NSObject, VideoSource {
                      buffer: sourcePixelBuffer,
                      orientation: videoOrientation,
                      forceReschedule: false)
+
+        // Hold on to the previous sample buffer to prevent tearing.
+        lastSampleBuffer2 = lastSampleBuffer
+        lastSampleBuffer = sampleBuffer
+    }
```

Comment:
This method is very big now.. > 100 lines. Can we factor some code out. 

Also adding more documentation, comments will be very useful.

Reply:
Input logic is now in processFrameInput.